testUtils/httpSpanBuilder.js

```diff
@@ -130,15 +132,17 @@ export class HttpSpanBuilder {
     this._span.info.httpInfo.host = host;
 
     const headers = HttpSpanBuilder.addHeader(this._span.info.httpInfo.request.headers, { host });
-    this._span.info.httpInfo.request.headers = headers;
+    this._span.info.httpInfo.request.headers = JSON.stringify(headers);
```

Comment:
so do we change the property types?

Reply:
yes - the BE can handle both.
The problem with the existing `payloadStringify` method is that it doesn't "remember" paths, so it is hard to refer to specific paths inside a big payload - we have to do it here

Reply:
I'm taking it back. I'll upload a fix the doesn't do it.